File: Exercícios sobre POO/Conta Banco 3-15/Conta Banco em python/conta.py
class SaldoInsuficienteException(Exception):
    "Saldo insuficiente"
    pass

class Conta_Bancaria:
    def __init__(self, titular, saldo) -> None:
        self.titular = titular
        self.saldo = float(saldo)
    
    def sacar(self, valor):
        try:
            if(self.saldo > valor):
                self.saldo -= valor
                print("Você sacou R$%.2f " % valor, end="")
                print("ficando com um saldo de R$%.2f" % self.saldo)
            else:
                raise SaldoInsuficienteException
        except SaldoInsuficienteException:
            print ("Saldo insuficiente")

    def depositar(self, valor):
        self.saldo += valor
        print("Você depositou R$%.2f " % valor, end="")
        print("ficando com um saldo de R$%.2f" % self.saldo)

File: Exercícios sobre POO/Conta Banco 3-15/Conta banco em java/src/Conta.java
public class Conta {
    private String titular;
    private double saldo;

    public Conta(String titular, double saldo){
        this.titular = titular;
        this.saldo = saldo;
    }

    public void sacar(double valor){
        try{
            if(this.saldo > valor){
                this.saldo -= valor;
                System.out.println("Voce sacou R$" + valor + "ficando com um saldo de R$" + this.saldo);
            }else{
                throw new SaldoInsuficienteException();
            }
        }catch(SaldoInsuficienteException ex){
            System.err.print(ex);
        }
    }
}
